src/refactor_nrepl/core.clj

```diff
@@ -95,7 +100,6 @@
   are pruned by this function."
   [pred dir]
   (->>  dir
-        io/file
```

Comment:
seems like it's not needed. this function is used with input coming from `dirs-on-classpath` which already returns `java.io.File`s.